### ISSUE TITLE
false positive

### ISSUE THREAD

**Issue by scafroglia93** [Clefspeare13/pornhosts#88]
eonline.com -> celebrity news portals
justjared.com -> celebrity news portals

Reference
https://github.com/nextdns/metadata/issues/159

**Comment by spirillen** [comment]
I can confirm all 3 domains from the link above

```
0.0.0.0/hosts:0.0.0.0 www.justjared.com
127.0.0.1/hosts:127.0.0.1 www.justjared.com
Mobile_0_0_0_0/hosts:0.0.0.0 m.justjared.com
SafeSearch/hosts:0.0.0.0 www.justjared.com

0.0.0.0/hosts:0.0.0.0 www.eonline.com
127.0.0.1/hosts:127.0.0.1 www.eonline.com
Mobile_0_0_0_0/hosts:0.0.0.0 m.eonline.com
SafeSearch/hosts:0.0.0.0 www.eonline.com

0.0.0.0/hosts:0.0.0.0 www.thehollywoodgossip.com
127.0.0.1/hosts:127.0.0.1 www.thehollywoodgossip.com
Mobile_0_0_0_0/hosts:0.0.0.0 m.thehollywoodgossip.com
SafeSearch/hosts:0.0.0.0 www.thehollywoodgossip.com
```
As I don't see any adult related content on these sites, I would't mind to remove them, But I'll awaits @Clefspeare13's opion for let's say a week before I'll take further action without approval

<details><summary>Screenshots</summary>
![image](https://user-images.githubusercontent.com/44526987/80541906-5912b580-89ac-11ea-90dc-f104094f9c33.png)

![image](https://user-images.githubusercontent.com/44526987/80541753-00dbb380-89ac-11ea-9efc-ace0d5e343d6.png)

![image](https://user-images.githubusercontent.com/44526987/80541841-2ec0f800-89ac-11ea-83d7-ce06c464f67c.png)
</details>

**Comment by Clefspeare13** [comment]
Thanks! Fixed in 9c2ba36
